### ISSUE TITLE
Keyboard navigation in search bar does not work

### ISSUE THREAD

**Issue by olafurpg** [scalacenter/scaladex#255]
While searching, pressing down+down+down+enter opens up the wrong project. For example, see this video recording:

[scaladex-keyboardnavigation.mov.zip](https://github.com/scalacenter/scaladex/files/429634/scaladex-keyboardnavigation.mov.zip)

In the video, the project scalamu is opened when I expected scalafmt to open.

**Comment by MasseGuillaume** [comment]
haha that mechanical keyboard.

thanks for repporting

**Comment by julienrf** [comment]
Hi, I can not reproduce the issue. Which browser and OS are you using?

**Comment by olafurpg** [comment]
I am unable to reproduce the error. Closing.
